### ISSUE TITLE
Images look rezzed when using Fast Image on iOS

### ISSUE THREAD

**Issue by banesrdjevic** [DylanVann/react-native-fast-image#739]
Fast Image works fine on Android, but the exact same images render rezzed and low quality in iOS. When I switch to the regular Image component the problem goes away, but then I lose the perks of fast image.

Can anyone tell me why this might be happening and if there's a solution for it?

Android:
![android](https://user-images.githubusercontent.com/6869891/97818091-f6092080-1c65-11eb-9a78-2dc1d7403887.jpg)

iOS:
![ios](https://user-images.githubusercontent.com/6869891/97818101-ff928880-1c65-11eb-8645-9c64df2b4ebc.png)

**Comment by banesrdjevic** [comment]
I'm pulling these images in from a remote url that has no file ending and returns base64.

**Comment by banesrdjevic** [comment]
Nothing yet, I've had to remove fast image from the iphone build
altogether. Still hoping to get an answer.

**Comment by Pietro-Putelli** [comment]
I’ve tried lots of libraries for caching images in react native, I even
tried to create one in my account, but it seems fast image works really
well apart this issue😕.

Anyway nice to talk with you, my name is Pietro from Italy.
What type of project are you working on?

**Comment by Pietro-Putelli** [comment]
I’ve tried lots of libraries for caching images in react native, I even
tried to create one in my account, but it seems fast image works really
well apart this issue😕.

**Comment by banesrdjevic** [comment]
I agree, and I've tried changing everything about the images as well.
Changed them from base64 to jpgs, changed the quality and size when the
user takes them. It'd be nice to know why this is happening to us and not
most other users of the library. If fast image had this issue universally
across all ios builds no one would use it.

**Comment by akeva001** [comment]
Having the same issue. Any solution to this?
